admin/src/pages/UserSearch.spec.js

```diff
@@ -121,8 +121,8 @@ describe('UserSearch', () => {
               searchText: '',
               currentPage: 1,
               pageSize: 25,
-              notActivated: null,
-              isDeleted: true,
+              filterByActivated: null,
```

Comment:
```suggestion
              filterByActivated: false,
```

Reply:
I like that we can see explizit that we switsch between `null` and `false`.
And then the logic works already if we include `true` in the future as well.